server/repository/src/db_diesel/report_query.rs

```diff
@@ -161,7 +161,7 @@ pub fn query_json(
     let rows = statement.raw_query();
     let rows = rows.mapped(|row| {
         let mut object = serde_json::Map::<String, serde_json::Value>::new();
-        for c in 0..column_names.len() {
+        for (c, _) in column_names.iter().enumerate() {
             let value = row.get_ref(c)?;
```

Comment:
Not sure about his one, but yes bit of an odd case here because effectively using the low level c API...